_translator-files/po/de/Include-Client-Commands.po

```diff
@@ -3,50 +3,51 @@
 # This file is distributed under the same license as the PACKAGE package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
 #
-#, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: \n"
+"Project-Id-Version: \n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: 2022-05-16 17:41+0200\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: de\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 3.0.1\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Client-Commands.md:2
 #, no-wrap
 msgid "`-M` or `--mutestream`  Starts Jamulus in muted state                                                      \n"
-msgstr ""
+msgstr "`-M` or `—mutestream` Startet Jamulus im stummen Zustand\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Client-Commands.md:3
 #, no-wrap
 msgid "`--mutemyown`  Mute me in my personal mix (\"headless\" mode only)                                                      \n"
-msgstr ""
+msgstr "`--mutemyown` Persönlichen Mix stummschalten (\"headless\" mode only)\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Client-Commands.md:4
 #, no-wrap
 msgid "`-c` or `--connect`  Connect to given server address on startup, format `address[:port]`  \n"
-msgstr ""
+msgstr "`-c` oder `--connect` Beim Start mit der angegebenen Serveradresse verbinden, Format `address[:port]`\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Client-Commands.md:5
 #, no-wrap
 msgid "`-j` or `--nojackconnect`  Disable auto JACK connections  \n"
-msgstr ""
+msgstr "`-j` oder `--nojackconnect` Automatische JACK-Verbindungen abschalten\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Client-Commands.md:6
 #, no-wrap
 msgid "`--ctrlmidich`  MIDI controller channel to listen on, control number offset and consecutive CC numbers (channels) and Mute Myself CC number. Format: `channel[;f*][;p*][;s*][;m*][;o]` See [Tips & Tricks](Tips-Tricks-More#using-ctrlmidich-for-midi-controllers) \n"
-msgstr ""
+msgstr "`—ctrlmidich` mIDI-Controller-Kanal, der abgehört werden soll, Offset der Kontrollnummer und fortlaufende CC-Nummern (Kanäle) und Mute Myself CC-Nummer. Format: kanal[;f*][;p*][;s*][;m*][;o]` Siehe [Tipps & Tricks](Tips-Tricks-More#Verwendung von ctrlmidich für MIDI-Controller)\n"
```

Comment:
```suggestion
msgstr "`—ctrlmidich` MIDI-Controller-Kanal, der abgehört werden soll, Offset der Kontrollnummer und fortlaufende CC-Nummern (Kanäle) und Mute Myself CC-Nummer. Format: kanal[;f*][;p*][;s*][;m*][;o]` Siehe [Tipps & Tricks](Tips-Tricks-More#Verwendung von ctrlmidich für MIDI-Controller)\n"
```

Please also check the link. I doubt spaces are ok. You probably need lower case headlines and `-` instead of ` `.